docs/nmt-lib.md

```diff
@@ -67,22 +67,22 @@ func (n *NamespacedMerkleTree) Push(namespacedData namespace.PrefixedData) error
 E.g.,
 
 ```go
-d := append(namespace.ID{0}, []byte("leaf_0")...) // the first `tree.NamespaceSize()` bytes of each data item is treated as its namespace ID.
+d := namespace.PrefixedData(append(namespace.ID{0}, []byte("leaf_0")...)) // the first `tree.NamespaceSize()` bytes of each data item is treated as its namespace ID.
```

Comment:
[optional] part of https://github.com/celestiaorg/nmt/issues/206

```suggestion
d := namespace.PrefixedData(append(namespace.ID{0}, []byte("leaf_0")...)) // the first `tree.NamespaceSize()` bytes of each data item is treated as its namespace.
```

Reply:
Done https://github.com/celestiaorg/nmt/pull/215/commits/1e0c14dfd41b8c1f2d73a003ba43975fca9ada51.